Tensile/KernelWriterAssembly.py

```diff
@@ -4164,7 +4173,13 @@ def graUnrollOffsets(self, kernel, tP):
               vgpr(v+s-1), "gro%s%s_%u_s%u"%(tP["tensorChar"], self.unrollChar, 0, s) )
         for l in range(1, tP["nru"]):
           # l>0, s=0
-          kStr += inst("_v_add_co_u32", vgpr(v+l*tP["glvw"]), self.vcc, stride, \
+          totalStride += stride
+          if kernel["DirectToVgpr%s"%tc] and lrvwOther == 2:
+            # DirectToVgpr + LocalReadVectorWidth=2 case, stride * 2 is added every 2. Add 1 in odd l case
+            totalStride = stride * (l - (l % 2)) + (l % 2)
```

Comment:
This might be a good place to rename "l", it is used in a bunch of equations here, and "1" and "l" look pretty similar in console fonts

Reply:
I understand l and 1 can be confusing...
My editor (also piece of code above) uses different colors for l and 1. Hence, it is not much confusing with these editors..
Using l (=lowercase of L) is not actually introduced by me....
(There are multiple functions using l)
Do you think I should change all of l in this file?
Changing only the above line looks not much organized.

Reply:
Yea, doesn't have to be part of this changeset, just highlighting it cause this'd be a good place to improve code clarity.

Reply:
OK. I can change it (in my next code change).
Do you have any good variable name for this on your mind?
Honestly, I do not like 1 character variable name.
So far, I am going to use 2 chars like ln, lt,